front-end/src/api/DashboardApi.js

```diff
@@ -9,3 +9,25 @@ export async function pullUserInfo(userID, token) {
   });
 }
 
+/*
+
+THIS HAS NOT BEEN TESTED
+
+export async function updateUserInfo(userID, token, email, twitter, phone) {
+  const url = process.env.NODE_ENV === 'development' ? 'http://localhost:3000' : 'https://githubconnector.herokuapp.com';
+  const config = {
+    headers: {
+    'Accept': 'application/json',
+    'Content-Type': 'application/json',
+    'Authorization': 'Bearer '  + token }
+  };
+  const payload = {
+    'email': email,
+    'twitter': twitter,
+    'phone': phone
+  };
+
+  return axios.put(`${url}/users/${userID}`, payload, config);
+}
+*/
```

Comment:
I'd be curious to hear from the others about what is best practice --
But I think if you still have code that's commented out it'd be best to just not include it in the commit / the pull request until it is finished.

But I'm not sure either, am curious what the others think!

Reply:
I'm not a big fan of commented out code in commits - From my current experience I notice that commented code that was meant to be removed in the future sometimes gets forgotten.

Reply:
Yeah, this probably should not have been added at all. I will remove it.